docs/src/how-to-use.md

```diff
@@ -7,20 +7,36 @@ Depth = 5
 
 ## Install
 
-* Clone the repository from [TulipaEnergyModel.jl](https://github.com/TulipaEnergy/TulipaEnergyModel.jl) into your local machine
-* Open the project in your favorite IDE (e.g., [Visual Studio Code](https://code.visualstudio.com/))
-* Start a Julia REPL and then:
-  * `]`: Enter package mode
-  * `activate .` : Activate here (root of project)
-  * `instantiate` : Instantiate any packages you need according to the Project.toml
-  * `test`: (Optional) Try running the tests to see if you're set up correctly - they should pass
+In Julia:
+
+- Enter package mode (press "]")
+
+```julia-pkg
+pkg> add TulipaEnergyModel
+```
+
+- Return to Julia mode (backspace)
+
+```julia
+julia> using TulipaEnergyModel
+```
+
+Optional (takes a minute or two):
+
+- Enter Tulipa package mode (press "]")
```

Comment:
The user won't be in the `TulipaEnergyModel` environment, because they are not in development mode. They can still test by using the explicit package name:

```suggestion
- Enter package mode (press "]")
```